web/controllers/screeningController/index.ts

```diff
@@ -405,4 +406,125 @@ export async function updateCourse(req: Request, res: Response) {
         return res.status(500).send("internal server error");
     }
 }
+/**
+ * @api {GET} /screening/instructors getInstructors
+ * @apiVersion 1.0.1
+ * @apiDescription
+ *
+ * Retrieves the first 20 courses that match the specified filters.
+ *
+ *
+ * Only screeners with a valid token in the request header can use the API.
+ *
+ * @apiName getInstructors
+ * @apiGroup Screener
+ *
+ * @apiUse Authentication
+ *
+ * @apiExample {curl} Curl
+ * curl -k -i -X GET -H "Token: <AUTHTOKEN>" [host]/api/screening/instructors
+ *
+ * @apiParam (URL Query) {string} screeningStatus get instructors with a certain screeningStatus
+ * @apiParam (URL Query) {string} search fuzzy search inside the instructors name and email
```

Comment:
Probably just add that the search will respect the Postgres Pattern matching characters. For example underscores are quite common in emails, so someone knows what the API does.